deploy/docker-compose/docker-compose.yml

```diff
@@ -110,13 +122,15 @@ services:
   dynamodb:
     image: amazon/dynamodb-local
     hostname: dynamodb
+    container_name: dynamo-xmidt
```

Comment:
does the `-xmidt` have a special meaning?

Reply:
I think to prevent clashes if you have any other dynamodb containers up for a different type of cluster.